File: base/src/main/java/cn/bluemobi/dylan/base/MyAppGlideModule.java
package cn.bluemobi.dylan.base;

import com.bumptech.glide.annotation.GlideModule;
import com.bumptech.glide.module.AppGlideModule;

/**
 * @author dylan
 * @date 2019-06-14
 */

@GlideModule
public class MyAppGlideModule extends AppGlideModule {
    @Override
    public boolean isManifestParsingEnabled() {
        return false;
    }
}

File: base/src/main/java/cn/bluemobi/dylan/base/MyAppGlideModule.kt
package cn.bluemobi.dylan.base;

import com.bumptech.glide.annotation.GlideModule;
import com.bumptech.glide.module.AppGlideModule;

/**
 * @author dylan
 * @date 2019-06-14
 */

@GlideModule
class MyAppGlideModule : AppGlideModule() {
    override fun isManifestParsingEnabled(): Boolean {
        return false
    }
}
